### ISSUE TITLE
Won't compile without CUDA

### ISSUE THREAD

**Issue by DarkShadow44** [bartoszek/AUR-luxcorerender#3]
I'm trying to compile with `DISABLE_CUDA=1 makepkg -fis`, but get the following error:
```
/tmp/ramdisk/luxcorerender/src/LuxCore-luxcorerender_v2.5beta1/src/slg/engines/oclrenderengine.cpp: In constructor ‘slg::OCLRenderEngine::OCLRenderEngine(const slg::RenderConfig*, bool)’:
/tmp/ramdisk/luxcorerender/src/LuxCore-luxcorerender_v2.5beta1/src/slg/engines/oclrenderengine.cpp:128:5: error: ‘CUDADeviceDescription’ was not declared in this scope
  128 |     CUDADeviceDescription *cudaDesc = (CUDADeviceDescription *)desc;
      |     ^~~~~~~~~~~~~~~~~~~~~
/tmp/ramdisk/luxcorerender/src/LuxCore-luxcorerender_v2.5beta1/src/slg/engines/oclrenderengine.cpp:128:28: error: ‘cudaDesc’ was not declared in this scope; did you mean ‘cudaDescs’?
  128 |     CUDADeviceDescription *cudaDesc = (CUDADeviceDescription *)desc;
      |                            ^~~~~~~~
      |                            cudaDescs
/tmp/ramdisk/luxcorerender/src/LuxCore-luxcorerender_v2.5beta1/src/slg/engines/oclrenderengine.cpp:128:63: error: expected primary-expression before ‘)’ token
  128 |     CUDADeviceDescription *cudaDesc = (CUDADeviceDescription *)desc;
      |                                                               ^

```

**Comment by bartoszek** [comment]
This should be fixed in [AUR/luxcorerender-git](https://aur.archlinux.org/packages/luxcorerender-git)
I'll backport the [patch](https://github.com/LuxCoreRender/LuxCore/commit/55b64c719a14957c40019bd51ded3ee4931e588d) here, otherwise we have to wait for `luxcore:2.5beta2` at least 🤭

**Comment by bartoszek** [comment]
Fixed in a58a5e4df6234422f3e7f1e1fbe7ba54b486c73d

**Comment by DarkShadow44** [comment]
Perfect, thanks!
